WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -220,6 +220,21 @@ protected SendControlViewModel(WalletService walletService, string title)
 					}
 				});
 
+			// Triggering the detection of same address values.
+			this.WhenAnyValue(x => x.Address)
+				.ObserveOn(RxApp.MainThreadScheduler)
+				.Subscribe(_ =>
+				{
+					this.RaisePropertyChanged(nameof(CustomChangeAddress));
+				});
+
+			this.WhenAnyValue(x => x.CustomChangeAddress)
+				.ObserveOn(RxApp.MainThreadScheduler)
+				.Subscribe(_ =>
+				{
+					this.RaisePropertyChanged(nameof(Address));
```

Comment:
This wasnt the recommendation in the comment you linked...

They said use ToProperty... Just use the setter its the simplest approach. Relying on the the scheduler to prevent recursion is a bad pattern.